conda-environment.yml

```diff
@@ -1,10 +1,13 @@
+# NOTE: the # [platform] selectors have no effect in conda environment files
+# however, do work in conda-forge feedstocks
```

Comment:
May be implemented sometime: https://github.com/conda/conda/issues/8089

Reply:
#TodayILearned. I assumed they would work the same way as in recipes, good to know

Reply:
BTW: I've now removed these deps because (annoyingly) they are needed in the system environment not the Cylc environment. Will come back to this one later...